Exiled.API/Features/Cassie.cs

```diff
@@ -124,5 +127,18 @@ public static void CustomSCPTermination(string scpname, DamageHandlerBase info)
         /// Clears the C.A.S.S.I.E queue.
         /// </summary>
         public static void Clear() => RespawnEffectsController.ClearQueue();
+
+        /// <summary>
+        /// Returns whether or not the given word is a valid C.A.S.S.I.E word.
+        /// </summary>
+        /// <param name="word">The word to check.</param>
+        /// <returns><see langword="true"/> if the word is valid, <see langword="false"/> otherwise.</returns>
```

Comment:
`<see langword="true"/> if the word is valid; otherwise, <see langword="false"/>.`